### ISSUE TITLE
[high] Color Contrast: Many Pages: White text against green background fails contrast ratio

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5445]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Measure the color contrast ratio of the white "Continue" text against green button background.
## Expected Result:
The contrast ratio of the text against the background should be at least 4.50:1 for small text and 3.00:1 for large text (14pt bold or 18 pt+).
## Actual Result:
This is a small text (13px=9.75pt). The white text (#FFFFFF) against green background (#03D47C) has a color contrast ratio of 1.96:1.

HTML:
`<div class="css-1dbjc4n" style="background-color: rgb(3, 212, 124); border-radius: 8px; border-width: 0px; height: 40px; justify-content: center; opacity: 1; padding-right: 12px; padding-left: 12px; -moz-box-pack: center;"><div dir="auto" class="css-901oao r-lrvibr" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 13px; font-weight: 700; text-align: center;">Continue</div></div>`

Other occurrences:
1. On Set password page the white "Set password" and "Pay" text (#FFFFFF) against green background (#03D47C) has a color contrast ratio of 1.96:1.
2. On Email verification page the white "Resend Link" and "Pay" text against green background (#03D47C) has a color contrast ratio of 1.96:1.
3. On Split bill overlay the white "Next" text against green button background (#03D47C) has a color contrast ratio of 1.96:1.
4. On Chat page the white "STG", "Save changes" and the amount value text against green background (#03D47C) have a color contrast ratio of 1.96:1.
5. On Request a call overlay the white "Call Me" text against green button background (#03D47C) has a color contrast ratio of 1.96:1.
6. On Create a new group overlay the white "Create Group" text against green button background (#03D47C) has a color contrast ratio of 1.96:1.
7. On Request money overlay the white "Next" text against green button background (#03D47C) has a color contrast ratio of 1.96:1.
## Workaround:
No workaround

## Area issue was found in:
Landing page, Set password page, Email verification page, Split bill overlay, Chat page, Request a call overlay, Create a new group overlay, Request money overlay.

## Failed WCAG checkpoints
1.4.3

## User impact:
The low-vision and color-blind users might not be able to read the text.

## Suggested resolution:
Use a darker shade of green for background to achieve a color contrast ratio of at least 4.50:1. An example of color to use is `#028850` which will give a color contrast ratio of 4.52:1 for the same white text.

## Platform:
Where is this issue occurring?
- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
![Bug5179530_Landing_page](https://user-images.githubusercontent.com/88733897/134511206-3ecb74b2-0574-4414-aa63-202e08c80e42.png)
![Bug5179530_Set_password](https://user-images.githubusercontent.com/88733897/134511228-a16e460c-e29c-4511-915c-907cc1b42a5d.png)
![Bug5179530_Email_verification](https://user-images.githubusercontent.com/88733897/134511243-27b9abc5-2a5f-483f-9809-5027afa731e1.png)
![Bug5179530_Split_bill](https://user-images.githubusercontent.com/88733897/134511263-841c995f-7c33-4b60-ae7e-364a5dc764fe.png)
![Bug5179530_Chat_and_Request_a_call_overlay](https://user-images.githubusercontent.com/88733897/134511310-07dfaca5-98ae-48c6-8eb6-1e7d5f668055.png)
![Bug5179530_Create_a_new_group](https://user-images.githubusercontent.com/88733897/134511335-6bd3a8e0-5ad7-4c6a-9f1a-12cf55eb5999.png)
![Bug5179530_Request_money](https://user-images.githubusercontent.com/88733897/134511354-ae9250b1-02ec-42ea-b964-cc25379089ed.png)

**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
